Nodejs/Product/TestAdapter/TestFrameworks/Jest/jest.js

```diff
@@ -116,5 +123,36 @@ function logError() {
     console.error.apply(console, errorArgs);
 }
 
+function readConfigs(projectFolder, context)
+{
+    // First look for Jest.config.js, otherwise look at the package.json under "jest" tag
+    var userConfig;
+    const jestConfigPath = projectFolder + "\\jest.config.js";
```

Comment:
I don't remember, how will this work if we have multiple projects in one solution? Do we get a new `projectFolder` for each project?

Reply:
Should `jest.config.ts` be considered as well?

Reply:
jest.config.ts failed on the tests because apparently it changes the way we can read the configs depending on the TS version. For now, if we think it's a good thing I suggest creating a new work item for it! (:

Reply:
A new work item for me makes sense. Lets merge this one first.